### PR TITLE
1.1.6

### PR DESCRIPTION
**Pull request by SamuelHall700**
# MeshiPhi Pull Request Template

Date: 16/02/2024
Version Number: 1.1.5 -> 1.1.6
 
## Description of change
Merging in Visual_iced dataloader, docs upates and additional features added when onboard the ship. Rebuilt the docs so as to be hosted correctly on GitHub

## Fixes # (issue)

# Testing
To ensure that the functionality of the MeshiPhi codebase remains consistent throughout the development cycle a testing strategy has been developed, which can be viewed in the document `test/testing_strategy.md`. 
This includes a collection of test files which should be run according to which part of the codebase has been altered in a pull request. Please consult the testing strategy to determine which tests need to be run. 

- [x] My changes have not altered any of the files listed in the testing strategy

- [ ] My changes result in all required regression tests passing without the need to update test files.  
  
> *list which files have been altered and include a pytest.txt file for each of
> the tests required to be run*
>
> The files which have been changed during this PR can be listed using the command

    git diff --name-only 1.1.x

- [ ] My changes require one or more test files to be updated for all regression tests to pass.   

> *include pytest.txt file showing which tests fail.*  
> *include reasoning as to why your changes cause these tests to fail.* 
>
> Should these changes be valid, relevant test files should be updated.  
> *include pytest.txt file of test passing after test files have been updated.*

# Checklist

- [x] My code follows [pep8](https://peps.python.org/pep-0008/) style guidelines.  
- [x] I have commented my code, particularly in hard-to-understand areas.  
- [x] I have updated the documentation of the codebase where required.  
- [x] My changes generate no new warnings.   
- [] My PR has been made to the `1.1.x` branch (**DO NOT SUBMIT A PR TO MAIN**)

### CODE REVIEW

File: docs/source/sections/Installation.rst [hunk 1]
```diff
@@ -6,27 +6,29 @@ In this section we will outline the installation steps for installing the softwa
 
 The first stage is installing a version of Python 3.9, if you don't have a working version. We suggest installing a working Anaconda distribution from https://www.anaconda.com/products/distribution#macos following the instructions on that page.
```
Comment (gecoombs) [inline docs/source/sections/Installation.rst:7]: Should this be edited to remove reference to anaconda/python 3.9? As here: https://github.com/antarctica/PolarRoute/blob/docs_update/docs/source/sections/Installation.rst?plain=1

**Review by gecoombs — CHANGES_REQUESTED** [review]
The committed contents of `docs/html/` don't match what you'd expect from the docs source. There are some old files in there but more importantly some whole sections are missing e.g. `docs/html/sections/Configuration/Configuration_overview.html`. Does it work for you locally @SamuelHall700? If so, this might be linked to having the docs folder in the .gitignore?

**Comment by SamuelHall700** [comment]
> The committed contents of `docs/html/` don't match what you'd expect from the docs source. There are some old files in there but more importantly some whole sections are missing e.g. `docs/html/sections/Configuration/Configuration_overview.html`. Does it work for you locally @SamuelHall700? If so, this might be linked to having the docs folder in the .gitignore? It looks like a similar story on `main` so this is probably also what caused the broken links we saw on the web version yesterday.

everything works locally, but `docs/html/*` being in the `.gitignore` file seemed to be the issue here. I've removed it and rebuilt the docs. It should hopefully work now :)

**Comment by gecoombs** [comment]
It all works much better now although I noticed the Background/Code Overview section is still written for PolarRoute. Maybe this section can just be removed from the MeshiPhi docs?

**Comment by SamuelHall700** [comment]
> It all works much better now although I noticed the Background/Code Overview section is still written for PolarRoute. Maybe this section can just be removed from the MeshiPhi docs?

I've removed this paragraph and rebuilt the docs

File: docs/html/sections/Plotting/mesh_plotting.html
Comment (gecoombs) [inline docs/html/sections/Plotting/mesh_plotting.html:1]: This section appears to be empty. There is an automodule statement but no content appears in the html file

File: README.md [hunk 2]
```diff
@@ -1,36 +1,40 @@
...
-<a href="https://antarctica.github.io/PolarRoute/"><img src="https://img.shields.io/badge/Manual%20-github.io%2FPolarRoute%2F-red" alt="Manual Page">
-<a href="https://pypi.org/project/polar-route/"><img src="https://img.shields.io/pypi/v/polar-route" alt="PyPi">
-<a href="https://github.com/antarctica/PolarRoute/tags"><img src="https://img.shields.io/github/v/tag/antarctica/PolarRoute" alt="Release Tag"></a>
-<a href="https://github.com/antarctica/PolarRoute/issues"><img src="https://img.shields.io/github/issues/antarctica/PolarRoute" alt="Issues"></a>
-<a href="https://github.com/antarctica/PolarRoute/blob/main/LICENSE"><img src="https://img.shields.io/github/license/antarctica/PolarRoute" alt="License"></a> -->
+<!-- <a href="https://colab.research.google.com/drive/12D-CN10X7xAcXn_df0zNLHtdiiXxZVkz?usp=sharing"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab" alt="Colab"> -->
+<a href="https://antarctica.github.io/MeshiPhi/"><img src="https://img.shields.io/badge/Manual%20-github.io%2FMeshiPhi%2F-red" alt="Manual Page"></a>
+
```
Comment (gecoombs) [inline README.md:7]: Is it an intentional change to have all of these buttons appear on separate lines?

Comment (gecoombs) [reply README.md]: Oh actually looks like this is a wider issue with github badges: https://github.com/badges/shields/issues/9978

**Review by gecoombs — APPROVED** [review]
Looks good

**Pull request merged**